test/integration/targets/filters/tasks/probe.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: a dummy variable to test the filter probe
+  set_fact:
+    some_var:
+      key1: begin
+      key2:
+        - foo
+        - bar
+        - subkey1: v1
+          subkey2: v2
+      key3: end
+
+- name: check that the value v2 is well reached
+  assert:
+    that:
+      - "some_var | probe('key2', 2, 'subkey2') == 'v2'"
+
+- name: check it's OK with a bad path and the default() filter
```

Comment:
You can do multiple asserts at the same time.
Also the quoting is not needed either.
But none of this is really important, having the tests is all that matters.